components/PeerReviews/PeerReviewRequestCard.tsx

```diff
@@ -0,0 +1,156 @@
+import { ReactElement, useState } from "react";
+import { PeerReviewRequest } from "~/config/types/peerReview";
+import { css, StyleSheet } from "aphrodite";
+import ALink from "~/components/ALink";
+import AuthorAvatar from "~/components/AuthorAvatar";
+import colors from "~/config/themes/colors";
+import { getUrlToUniDoc } from "~/config/utils/routing";
+import { timeSince } from "~/config/utils/dates";
+import DropdownButton from "~/components/Form/DropdownButton";
+import { ID } from "~/config/types/root_types";
+import PeerReviewRequestModal from "./PeerReviewInviteModal";
+import PeerReviewPerson from "./PeerReviewPerson";
+
+
+type Props = {
+  peerReviewRequest: PeerReviewRequest;
+  fetchReviewRequests: Function;
+};
+
+function PeerReviewRequestCard({
+  peerReviewRequest,
+  fetchReviewRequests,
+}: Props): ReactElement {
+
+  const opts = [{
+    label: "Invite",
+    value: "invite"
+  }, {
+    label: "Decline",
+    value: "decline"
+  }, {
+    label: "Close",
+    value: "close"    
+  }];
```

Comment:
Let's move this outside of component. No need for it to be part of it

Reply:
Yeah makes sense